src/tike/operators/convolution.py

```diff
@@ -6,11 +6,44 @@
 
 
 class Convolution(Operator):
-    """2D Convolution operator with linear interpolation."""
+    """A 2D Convolution operator with linear interpolation.
+
+    Compute the product two arrays at specific relative positions.
+
+    Attributes
+    ----------
+    nscan : int
+        The number of scan positions at each angular view.
+    fly : int
+        The number of consecutive scan positions that describe a fly scan.
+    nmode : int
+        The number of probe modes per scan position.
+    probe_shape : int
+        The pixel width and height of the (square) probe illumination.
+    nz, n : int
+        The pixel width and height of the reconstructed grid.
+    ntheta : int
+        The number of angular partitions of the data.
+
+    Parameters
+    ----------
+    psi : (ntheta, nz, n) complex64
+        The complex wavefront modulation of the object.
```

Comment:
The complex wavefront at object plane

Reply:
It's not that. It's the array containing the refractive indices of the object. Maybe I should say "The refractive indices of the object (unitless)"?